src/App.css

```diff
@@ -153,3 +153,30 @@ input {
   background-color: #EA0385;
   font-weight: 600;
 }
+
+.splash{
+  cursor: pointer;
```

Comment:
some things like the profile photo and text input field show up above this splash so you can add `z-index: 1;` or something similar to fix that